molscore/manager.py

```diff
@@ -1001,6 +974,8 @@ def score(
         ), "No molecular representations provided, please supply smiles or other representations as keyword arguments"
         if smiles:
             molecular_inputs["smiles"] = smiles
+            if not mol_ids:
```

Comment:
Not without `canonicalise=True`, this is redundant with setting `smiles` as `mol_ids` later?